enclave_wrangler/config.py

```diff
@@ -77,7 +77,7 @@
 # todo: All indexes should be listed here, and initialize/refresh refactored such that indexex get created by generating
 #  SQL from here, rather than reading from the DDL files.
 # Ordered because of transformation dependencies
-FAVORITE_DATASETS = OrderedDict({
```

Comment:
@Sigfried Finally renamed `FAVORITE_DATASETS` to `DATASET_REGISTRY`. Makes more sense now I think. We now register information about a number of datasets, and rather than downloading all of our 'favorite' ones at once, we typically select them based on groupings now.